database/sql/jobs.go

```diff
@@ -84,7 +85,8 @@ func (s *sqlDatabase) paramsJobToWorkflowJob(ctx context.Context, job params.Job
 	if job.RunnerName != "" {
 		instance, err := s.getInstanceByName(s.ctx, job.RunnerName)
 		if err != nil {
-			slog.With(slog.Any("error", err)).ErrorContext(ctx, "failed to get instance by name")
+			// This usually is very normal as not all jobs run on our runners.
+			slog.DebugContext(ctx, fmt.Sprintf("failed to get instance by name: %s", job.RunnerName))
```

Comment:
Let's try something like this:

```suggestion
			slog.With(slog.Any("error", err)).DebugContext(ctx, "failed to get instance by name", "instance_name", job.RunnerName)
```

slog allows us to have structured logging, where we can add keys and values to those keys. No need to use `fmt` to format the messages.

in the suggestion above, if you export the log in json format, you will have the message `failed to get instance by name` and 2 keys: `error: "error message goes here"` and `instance_name: "garm-XXXXX"`. When printing as text, they'll show up as `error=message` and `instance_name=garm-XXXX`.

Reply:
.. okay, actually I was aiming to remove the word 'error' from those log lines, I usually grep for error and this is not really an error ..even if there is a error-struct  thrown somewhere. Of course, I can change my grepping .. but do you see it's valuable to see the error message ?

Reply:
In this context, I think we can just remove it. It's just spam.